app/jsx/components/TabAuthorComp.jsx

```diff
@@ -16,31 +17,104 @@ class CitationComp extends React.Component {
+    let lastCrumb = (p.header && p.header.breadcrumb) ? p.header.breadcrumb[p.header.breadcrumb.length-1] : null
+    let [journal_name, volume, issue, issue_url] = [null, null, null, null]
     if (p.attrs['ext_journal']) {
-      issn = p.attrs['ext_journal'] && p.attrs['ext_journal']['issn']
-    } 
-    let peer_reviewed = p.attrs['is_peer_reviewed'] ? "True" : "False",
-        permalink = "https://escholarship.org/uc/item/" + p.id,
-        ezid = "https://ezid.cdlib.org/id/ark:/13030/qt" + p.id,
-        appearsIn = p.appearsIn.map(function(node, i) {
-          return ( <span key={i}><Link to={"/uc/"+node.id} className="o-textlink__secondary">{node.name}</Link><br/></span> )
-        }),
-        retrieved_suffix = ''
+      let extj = p.attrs['ext_journal']
+      journal_name = extj['name'] 
+      volume = extj['volume']
+      issue = extj['issue'] 
+    } else if (p.citation) {
+      let c = p.citation 
+      // 'journal_name' defined by 'appears_in' value below
+      volume = c['volume'] ? c['volume'] : ''
+      issue = c['issue'] 
+      issue_url = lastCrumb ? lastCrumb.url : null 
+    }
+    let issue_title = lastCrumb ? lastCrumb.name : null
+    let journal_stmnt
+    if (journal_name || volume || issue) {
+      let voliss = this.customIssueTitle(volume, issue, p.numbering, issue_title) 
+      journal_stmnt = journal_name ? journal_name+", "+voliss : voliss
+    }
+
+    let issn = p.attrs['ext_journal'] && p.attrs['ext_journal']['issn']
+
+    let unit_type
+    if (p.unit) { unit_type = p.unit.type == 'journal' ? "Journal" : "Series" }
+
+    let permalink = "https://escholarship.org/uc/item/" + p.id
+    let appearsIn = p.appearsIn.map(function(node, i) {
+      return ( <span key={i}><Link to={"/uc/"+node.id} className="o-textlink__secondary">{node.name}</Link><br/></span> )
+    })
+
+    let doi
+    if (p.attrs['doi']) { doi = p.attrs['doi'].startsWith("http") ? p.attrs['doi'] : "https://doi.org/" + p.attrs['doi'] }
+
+    let retrieved_suffix = ''
     if (p.attrs['orig_citation']) {
       let url = window.location.href.split('#')[0],
           date = p.formatDate(),
           r = "Retrieved " + date + ", from " 
       retrieved_suffix = <span>{r}<Link to="url">{url}</Link></span>
     }
+
+    // Local ID looks something like this:
+    //   [{"id": "Southern_ucsc_0036E_11162", "type": "proquest"}, {"id": "http://dissertations.umi.com/ucsc:11162", "type": "other"}, {"id": "ark:/13030/m5dn8t2h", "type": "merritt"}] 
+    let localIdLabel = type => {
+      let v = {
+        'arXiv':        "arXiv ID: ",
+        'lbnl':         "LBNL Report #: ",
+        'merritt':      "Merritt ID: ",
+        'oa_harvester': "UCPMS ID: ",
+        'other':        "",
+        'pmid':         "Pubmed ID: ",
+        'pmcid':        "PubMed Central ID: ",
+        'proquest':     "ProQuest ID: ",
+        'repec':        "RePEc ID: ",
+        'default':      ""
+      }
+      return type ? v[type] : v["default"]
```

Comment:
Note that this will probably break when we add types in the future. It'd be safer to `return v[type] ? v[type] : v["default"]` I think.